src/backend/joanie/payment/admin.py

```diff
@@ -194,6 +203,20 @@ def expiration_date(credit_card):
 
         return f"{credit_card.expiration_month:02d}/{credit_card.expiration_year:02d}"
 
+    @staticmethod
+    def has_initial_issuer_transaction_identifier(credit_card):
+        """
+        Return the initial issuer transaction identifier
+        """
+        return "OK" if credit_card.initial_issuer_transaction_identifier else "-"
+
+    @staticmethod
+    def has_token(credit_card):
+        """
+        Return the token of the credit card
+        """
+        return "OK" if credit_card.token else "-"
+
```

Comment:
<img width="765" alt="image" src="https://github.com/openfun/joanie/assets/9265241/af9b5aee-45e3-4218-9d1d-7d4865de10c1">

```suggestion
    @staticmethod
    @admin.display(boolean=True)
    def has_initial_issuer_transaction_identifier(credit_card):
        """
        Return the initial issuer transaction identifier
        """
        return credit_card.initial_issuer_transaction_identifier is not None

    @staticmethod
    @admin.display(boolean=True)
    def has_token(credit_card):
        """
        Return the token of the credit card
        """
        return credit_card.token is not None

```

Reply:
Oh thanks ! I didn't find a way to to that :pray: